### PR TITLE
Release 0.2.1

### PR DESCRIPTION
**Pull request by cournape**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 MAJOR = 0
 MINOR = 2
-MICRO = 1
+MICRO = 2
 
 IS_RELEASED = False
```
Comment (johntyree) [inline setup.py]: `IS_RELEASED` ?

Comment (cournape) [inline setup.py]: yes ?

Comment (cournape) [inline setup.py]: oh, I think I see what you mean. With Simon, we figured it was very easy to forget to switch back `IS_RELEASED` to False, so we do the flip back and forth in one PR

Comment (johntyree) [inline setup.py]: I'm confused. You mean this tagged commit is not what's released? You release it with this flag flipped and then flip it back?

Comment (cournape) [inline setup.py]: yes, that's why there are 2 commits

Comment (johntyree) [inline setup.py]: Somehow I didn't notice that. OK Sounds good :)

**Pull request merged**